db/schema.rb

```diff
@@ -223,7 +223,7 @@
     t.boolean "passporting_benefit"
     t.string "benefit_type"
     t.string "has_benefit_evidence"
-    t.index ["crime_application_id"], name: "index_people_on_crime_application_id", unique: true
+    t.index ["type", "crime_application_id"], name: "index_people_on_type_and_crime_application_id", unique: true
```

Comment:
I keep getting this diff when I migrate, is anyone else getting this?

Reply:
@hiboabd delete your dev database. I think you have this from running migrations in our partner spike.